### PR TITLE
[CosmosDB]Update SDK and Add PublicNetworkAccess to New/Update-AzCosmosDBAccount

### PR DESCRIPTION
**Pull request by soeom**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/CosmosDB/CosmosDB/CosmosDBAccount/NewAzCosmosDBAccount.cs [hunk 1]
```diff
@@ -85,6 +85,10 @@ public class NewAzCosmosDBAccount : AzureCosmosDBCmdletBase
         [PSArgumentCompleter("Sql", "MongoDB", "Gremlin", "Cassandra", "Table")]
         public string ApiKind { get; set; }
 
+        [Parameter(Mandatory = false, HelpMessage = Constants.PublicNetworkAccessHelpMessage)]
+        [ValidateNotNullOrEmpty]
+        public string PublicNetworkAccess { get; set; }
```
Comment (MehaKaushik) [inline src/CosmosDB/CosmosDB/CosmosDBAccount/NewAzCosmosDBAccount.cs:90]: Are the values of this string fixed ?? If yes, then use PSArgumentCompleter, (see ApiKind for reference)

File: src/CosmosDB/CosmosDB/CosmosDBAccount/UpdateAzCosmosDBAccount.cs [hunk 2]
```diff
@@ -123,6 +127,10 @@ public override void ExecuteCmdlet()
             {
                 databaseAccountUpdateParameters.DisableKeyBasedMetadataWriteAccess = DisableKeyBasedMetadataWriteAccess;
             }
+            if (DisableKeyBasedMetadataWriteAccess != null)
```
Comment (MehaKaushik) [inline src/CosmosDB/CosmosDB/CosmosDBAccount/UpdateAzCosmosDBAccount.cs]: if (PublicNetworkAccess != null)

File: src/CosmosDB/CosmosDB/help/New-AzCosmosDBAccount.md [hunk 3]
```diff
@@ -291,6 +291,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -PublicNetworkAccess
```
Comment (MehaKaushik) [inline src/CosmosDB/CosmosDB/help/New-AzCosmosDBAccount.md:294]: Did you generate these using Update-MarkdownHelp cmdlet ? If not, please do so, that might fix the build error in the CI.

**Comment by MehaKaushik** [comment]
Left some comments, please have a look.

File: src/CosmosDB/CosmosDB/help/New-AzCosmosDBAccount.md (hunk 3 above)
Comment (soeom) [reply src/CosmosDB/CosmosDB/help/New-AzCosmosDBAccount.md:295]: Gotcha Thanks

File: src/CosmosDB/CosmosDB/CosmosDBAccount/UpdateAzCosmosDBAccount.cs (hunk 2 above)
Comment (soeom) [reply src/CosmosDB/CosmosDB/CosmosDBAccount/UpdateAzCosmosDBAccount.cs]: I actually caught the typo beforehand and fixed it with it2

File: src/CosmosDB/CosmosDB/CosmosDBAccount/NewAzCosmosDBAccount.cs (hunk 1 above)
Comment (soeom) [reply src/CosmosDB/CosmosDB/CosmosDBAccount/NewAzCosmosDBAccount.cs:90]: added PSArgumentCompleter. Thanks

File: src/CosmosDB/CosmosDB/help/Update-AzCosmosDBCassandraKeyspaceThroughput.md [hunk 4]
```diff
@@ -32,6 +32,8 @@ Update-AzCosmosDBCassandraKeyspaceThroughput [-Name <String>] -Throughput <Int32
  [<CommonParameters>]
 ```
 
+## DESCRIPTION
```
Comment (MehaKaushik) [inline src/CosmosDB/CosmosDB/help/Update-AzCosmosDBCassandraKeyspaceThroughput.md:35]: Revert this.

File: src/CosmosDB/CosmosDB/help/Update-AzCosmosDBCassandraTableThroughput.md [hunk 5]
```diff
@@ -33,6 +33,8 @@ Update-AzCosmosDBCassandraTableThroughput [-Name <String>] -Throughput <Int32>
  [<CommonParameters>]
 ```
 
+## DESCRIPTION
```
Comment (MehaKaushik) [inline src/CosmosDB/CosmosDB/help/Update-AzCosmosDBCassandraTableThroughput.md:36]: Revert

File: src/CosmosDB/CosmosDB/help/Update-AzCosmosDBGremlinDatabaseThroughput.md [hunk 6]
```diff
@@ -32,6 +32,8 @@ Update-AzCosmosDBGremlinDatabaseThroughput [-Name <String>] -Throughput <Int32>
  [<CommonParameters>]
 ```
 
+## DESCRIPTION
```
Comment (MehaKaushik) [inline src/CosmosDB/CosmosDB/help/Update-AzCosmosDBGremlinDatabaseThroughput.md:35]: Revert

File: src/CosmosDB/CosmosDB/help/Update-AzCosmosDBGremlinGraphThroughput.md [hunk 7]
```diff
@@ -33,6 +33,8 @@ Update-AzCosmosDBGremlinGraphThroughput [-Name <String>] -Throughput <Int32>
  [<CommonParameters>]
 ```
 
+## DESCRIPTION
```
Comment (MehaKaushik) [inline src/CosmosDB/CosmosDB/help/Update-AzCosmosDBGremlinGraphThroughput.md:36]: Revert

File: src/CosmosDB/CosmosDB/help/Update-AzCosmosDBMongoDBCollectionThroughput.md [hunk 8]
```diff
@@ -33,6 +33,8 @@ Update-AzCosmosDBMongoDBCollectionThroughput [-Name <String>] -Throughput <Int32
  [<CommonParameters>]
 ```
 
+## DESCRIPTION
```
Comment (MehaKaushik) [inline src/CosmosDB/CosmosDB/help/Update-AzCosmosDBMongoDBCollectionThroughput.md:36]: Revert

File: src/CosmosDB/CosmosDB/help/Update-AzCosmosDBMongoDBDatabaseThroughput.md [hunk 9]
```diff
@@ -32,6 +32,8 @@ Update-AzCosmosDBMongoDBDatabaseThroughput [-Name <String>] -Throughput <Int32>
  [<CommonParameters>]
 ```
 
+## DESCRIPTION
```
Comment (MehaKaushik) [inline src/CosmosDB/CosmosDB/help/Update-AzCosmosDBMongoDBDatabaseThroughput.md:35]: Revert

File: src/CosmosDB/CosmosDB/help/Update-AzCosmosDBSqlContainerThroughput.md [hunk 10]
```diff
@@ -33,6 +33,8 @@ Update-AzCosmosDBSqlContainerThroughput [-Name <String>] -Throughput <Int32>
  [<CommonParameters>]
 ```
 
+## DESCRIPTION
```
Comment (MehaKaushik) [inline src/CosmosDB/CosmosDB/help/Update-AzCosmosDBSqlContainerThroughput.md:36]: Revert

File: src/CosmosDB/CosmosDB/help/Update-AzCosmosDBSqlDatabaseThroughput.md [hunk 11]
```diff
@@ -31,6 +31,8 @@ Update-AzCosmosDBSqlDatabaseThroughput [-Name <String>] -Throughput <Int32>
  [<CommonParameters>]
 ```
 
+## DESCRIPTION
```
Comment (MehaKaushik) [inline src/CosmosDB/CosmosDB/help/Update-AzCosmosDBSqlDatabaseThroughput.md:34]: Revert

File: src/CosmosDB/CosmosDB/help/Update-AzCosmosDBTableThroughput.md [hunk 12]
```diff
@@ -30,6 +30,8 @@ Update-AzCosmosDBTableThroughput [-Name <String>] -Throughput <Int32> -InputObje
  [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
+## DESCRIPTION
+
```
Comment (MehaKaushik) [inline src/CosmosDB/CosmosDB/help/Update-AzCosmosDBTableThroughput.md:34]: Revert

**Comment by MehaKaushik** [comment]
@msJinLei  looks good to me, approved.

**Pull request merged**